bookmarks/templates/settings/about.html

```diff
@@ -0,0 +1,19 @@
+{% extends "bookmarks/layout.html" %}
+
+{% block content %}
+    <div class="settings-page">
+
+        {% include 'settings/nav.html' %}
+
+        {# Export about #}
+        <section class="content-area">
```

Comment:
Adding a new page / tab for this is a bit much. I think we should add it as a new section to the general settings page (https://github.com/sissbruecker/linkding/blob/master/bookmarks/templates/settings/general.html), below the existing sections.